src/CommunityToolkit/Xamarin.CommunityToolkit/Extensions/ViewExtension .shared.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.CommunityToolkit.UI.Views.Options;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Extensions
+{
+	public static class ViewExtension
```

Comment:
Looks like there is a space on the FileName, can you check it?

Reply:
Yes, but it was fixed in commit https://github.com/xamarin/XamarinCommunityToolkit/pull/846/commits/6ba8d7f6984e593b671afde4d64b82ea107ef694